src/test/java/pl/coderslab/projectwybankuj/service/BankServiceTest.java

```diff
@@ -0,0 +1,83 @@
+package pl.coderslab.projectwybankuj.service;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.web.multipart.MultipartFile;
+import pl.coderslab.projectwybankuj.entity.Bank;
+import pl.coderslab.projectwybankuj.repository.BankRepository;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+class BankServiceTest {
+
+    private BankRepository bankRepositoryMock = Mockito.mock(BankRepository.class);
+    private HttpServletRequest requestMock = Mockito.mock(HttpServletRequest.class);
+    private BankService bankService = new BankService(bankRepositoryMock, requestMock);
+
+    @Test
+    void shouldFindAllBanks() {
+        // when
+        List<Bank> foundBanks = (List<Bank>) bankService.findAllBanks();
+
+        // then
+        Mockito.verify(bankRepositoryMock).findAll();
```

Comment:
Testy z verify są spoko jeżeli testujemy coś co zwraca void (tak jak było w przypadku email) tutaj jednak testujesz coś co coś zwraca a więc możesz zamockować repozytorium tak aby coś zwracało i spradzić czy serwis też to zwróci